mungegithub/mungers/submit-queue.go

```diff
@@ -67,6 +67,13 @@ var (
 	_ = fmt.Print
 	// This MUST cause a RETEST of everything in the sq.RequiredRetestContexts
 	retestBody = fmt.Sprintf("@%s test this [submit-queue is verifying that this PR is safe to merge]", jenkinsBotName)
+
+	requiredContexts = []string{
```

Comment:
Actually this is fine, i'm just a little worried/uncomfortable about the usage in the 'required-retest-contexts' flag...  I want that part broken out somehow...